### PR TITLE
Test

### PR DESCRIPTION
**Pull request by AntjeWalter**
testi huhu

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default function Home({
   onSorting,
 }) {
   const { data: session } = useSession();
+  console.log("testi");
```
Comment (marvinRoellecke) [inline pages/index.js:22]: please remove console log

Comment (AntjeWalter) [reply pages/index.js:22]: no

Comment (marvinRoellecke) [reply pages/index.js:22]: doch

Comment (marvinRoellecke) [reply pages/index.js:22]: now!

**Pull request closed without merging**